### ISSUE TITLE
Compatibility for Python 3.6.x

### ISSUE THREAD

**Issue by wenh06** [alexa/massive#6]
`str.isascii` was added in Python 3.7. For compatibility for Python 3.6.x, can you consider using
```python
def isascii(s: str) -> bool:
    try:
        return s.isascii()
    except AttributeError:
        return all([ord(c) < 128 for c in s])
```

**Comment by jgmf-amazon** [comment]
Hi @wenh06 , good call. We will make this update soon and post a PR. Thanks.

**Comment by jgmf-amazon** [comment]
Thanks, @wenh06 , that code change looks good. Would you like to do a PR? Otherwise, we'll make the change.

**Comment by wenh06** [comment]
> Thanks, @wenh06 , that code change looks good. Would you like to do a PR? Otherwise, we'll make the change.

It is the last commit of PR #5, which is automatically appended. I don't know how to split PR #5 into two PRs.

**Comment by cperiz** [comment]
This was addressed via #10
